cosmos/bin/cosmos

```diff
@@ -308,6 +328,9 @@ if not ARGV[0].nil? # argument(s) given
   when 'rake'
     puts `rake #{ARGV[1..-1].join(' ')}`
 
+  when 'validate'
```

Comment:
I was thinking "dryrun" when I read this, but I think validate is ok too.  Thoughts?

Reply:
I think validate is clearer plus that's what someone else used to describe this.